src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryTable.cs

```diff
@@ -164,6 +164,20 @@ public static FormulaValue FilterTable(EvalVisitor runner, SymbolContext symbolC
             return new InMemoryTableValue(irContext, rows);
         }
 
+        public static FormulaValue IndexTable(IRContext irContext, FormulaValue[] args)
+        {
+            var arg0 = (TableValue)args[0];
+            var arg1 = (NumberValue)args[1];
+            int rowIndex = (int)arg1.Value;
+
+            if (rowIndex < 1 || rowIndex > arg0.Rows.Count())
+            {
+                return CommonErrors.ArgumentOutOfRange(irContext);
+            }
+
+            return arg0.Rows.ElementAtOrDefault(rowIndex - 1).ToFormulaValue();
```

Comment:
Why not just start indexing at 0?

Reply:
Hi Emily, it's because to get the first row in Excel's INDEX function we provide an index of 1. An index of 0 would return the entire table as is.

Reply:
Power Fx (because of its Excel) starts all indices at 1. Mid("Hello", 2) = "ello", not "llo"; our current way to implement Index is by using `Last(FirstN(<table>, <index>))`, and that is also 1-based

Reply:
Got it, thanks!